### ISSUE TITLE
The Clear Control button truly clears the app 

### ISSUE THREAD

**Issue by jacobwod** [hajkmap/Hajk#727]
Currently, the _Clear_ button is actually a "Hide all LayerSwitcher layers, except the background layer"-button.

I've heard opinions that we should change the way it works and that it should really clear as much as possible. For example:
- anything from the Measure plugin
- Draw plugin
- any active searches, including spatial searches and search results list
- Edit plugin
- close all visible windows

**What are the opinions on this?**

**Comment by sweco-sesqqq** [comment]
Could it be an idea to have a drop-down-list under the clear button? Giving the users the possibility to:

- Clear the entire map (As above)
- Close all layers
- ...

**Comment by jacobwod** [comment]
Absolutely. With a reasonable default, as I doubt that the majority of users will ever use the advanced options. But we should implement it like that and save the current settings between sessions, so a user can set his own preferences.

**Comment by sweco-sesqqq** [comment]
Yes, saving the preferred options between sessions sounds like a good idea!
